definitions/private-law/json/CaseEvent/CaseEvent.json

```diff
@@ -997,7 +997,7 @@
     "Description": "Transfer to another court",
     "DisplayOrder": 107,
     "PreConditionState(s)": "CASE_ISSUED;JUDICIAL_REVIEW",
-    "PostConditionState": "*",
+    "PostConditionState": "PROCEEDS_IN_HERITAGE_SYSTEM(cantFindCourtCheck!=\"null\"):1;*",
```

Comment:
can we do this via backend call please with about to submit callback?

Reply:
checking if we can do it in backend

Reply:
anything on this?

Reply:
Moved this logic to backend call